src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+const people = [];
+
+function sortList() {
+  list
+    .forEach(item => people.push({
+      name: item.innerText,
+      ...item.dataset,
+    }));
+
+  people.sort((a, b) => convert(b.salary) - convert(a.salary));
+
+  function convert(str) {
+    return +str.replace(/\D/g, '');
+  }
+}
```

Comment:
Function sortList should sort the list by salary from data attributes, not from objects properties
![image](https://user-images.githubusercontent.com/30314756/135398638-ef8cbeb1-427d-4c2c-9cd8-0dc82a16bc3f.png)